src/or/command.c

```diff
@@ -473,11 +521,24 @@ command_process_relay_cell(cell_t *cell, channel_t *chan)
               "unknown circuit %u on connection from %s. Dropping.",
               (unsigned)cell->circ_id,
               channel_get_canonical_remote_descr(chan));
+
+    /* PrivCount will just ignore this, but do it anyway for consistency */
```

Comment:
Note that `control_event_privcount_circuit_cell ` should only be called in this function in places where this function returns before calling `circuit_receive_relay_cell`; if `circuit_receive_relay_cell` is called, the cell event will be emitted there instead.